samples/Sentry.Samples.GenericHost/Sentry.Samples.GenericHost.csproj

```
@@ -12,15 +12,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="../../src/Sentry.Extensions.Logging/Sentry.Extensions.Logging.csproj" />
+    <ProjectReference Include="../../src/Sentry.AspNetCore/Sentry.AspNetCore.csproj" />
```

Do we need to change the dependency to ASP.NET Core integration? The new types are all on the S.E.L, right?
```suggestion
    <ProjectReference Include="../../src/Sentry.Extensions.Logging/Sentry.Extensions.Logging.csproj" />
```

---

New types are not on S.E.L. right now, see comment below